README.md

```diff
@@ -90,6 +90,7 @@ If you wish to view the [nextstrain-maintained narratives](https://nextstrain.or
 ### Run auspice
 
 ```bash
+auspice build --verbose
```

Comment:
FYI - the `prepare` script was so that installs of auspice via `npm install --global .` (line 58 of this README) would leave the user with a built instance of auspice they could run immediately with `auspice view ...`. I'll double check that this is still the case (I expect it is). 

I do agree with the intention of this PR tho -- would be nice to not have time-consuming build steps occurring when we don't want them.

Reply:
>the `prepare` script was so that installs of auspice via `npm install --global .` would leave the user with a built instance of auspice they could run immediately with `auspice view ...`.

Thanks for the context – you're right, this is still necessary. Also makes me think about some side effects of #1608, especially for anyone using `npm install --global auspice`. I'll probably create another PR or start a discussion about the effects of these two PRs.

Reply:
(I think you realised my typo Victor, but I was actually referring to line 50 `npm install --global auspice` -- the dev install separates out the build step for clarity.)